### PR TITLE
Fix wxWebViewWebKit build with WebKit2 < 2.8

### PR DESCRIPTION
**Pull request by vadz**
This allows this code to be compiled with WebKit2 2.6 included in Debian
Jessie.

---

I've basically fixed it blindly just to make it compile, it would be nice if somebody who actually knew anything about WebKit2 could look at these changes and confirm that they make sense.

### CODE REVIEW

File: src/gtk/webview_webkit2.cpp [hunk 1]
```diff
@@ -950,7 +950,11 @@ bool wxWebViewWebKit::IsBusy() const
 
 void wxWebViewWebKit::SetEditable(bool enable)
 {
+#if WEBKIT_CHECK_VERSION(2, 8, 0)
     webkit_web_view_set_editable(m_web_view, enable);
+#else
+    g_object_set(m_web_view, "editable", enable ? TRUE : FALSE, NULL);
```
Comment (swt2c) [inline src/gtk/webview_webkit2.cpp]: It looks like the `editable` property is 2.8+ also:
https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WebKitWebView--editable

Looks like setting editable may not be possible on < 2.8.

**Review by swt2c — COMMENTED** [review]
The other changes look correct.

File: src/gtk/webview_webkit2.cpp (hunk 1 above)
Comment (vadz) [inline src/gtk/webview_webkit2.cpp]: Thanks, I didn't notice this. Shouldn't we make `SetEditable()` return bool to indicate whether it managed to make the window editable? Using `IsEditable()` to check for the other function success looks a bit weird... But for now I guess I'll just make `IsEditable()` always return false for pre-2.8 versions.

Comment (swt2c) [inline src/gtk/webview_webkit2.cpp]: Yes, that does make sense, but I guess the other platforms can always enable editing unconditionally, so the API was not made to anticipate failure.

**Pull request closed without merging**